src/site/topic12.rst

```diff
@@ -8,6 +8,43 @@ Idea #1
 Idea #2
 =======
 
+* Use an array for the container
+* Keep track of the ``front`` index
+* Keep track of the ``rear`` index
+* Size will conveniently be ``rear - front``
```

Comment:
Not always true. Consider any cases where `rear < front`.

Reply:
Or, it seems this is a toy implementation meant as a stepping stone to cyclic indexes. Fair enough.